dcos/subcommand.py

```diff
@@ -323,6 +322,24 @@ def uninstall(package_name):
 BIN_DIRECTORY = 'Scripts' if util.is_windows_platform() else 'bin'
 
 
+def _find_virtualenv(bin_directory):
+    """
+    :param bin_directory: directory to first use to find virtualenv
+    :type bin_directory: str
+    :returns: Absolute path to virutalenv program
+    :rtype: str
+    """
+
+    virtualenv_path = os.path.join(bin_directory, 'virtualenv')
+    if not os.path.exists(virtualenv_path):
+        virtualenv_path = util.which('virtualenv')
+
+    if virtualenv_path is None:
+        raise DCOSException('Unable to find the virtualenv program')
```

Comment:
This is finding virtualenv. We need virtualenv for installing the subcommands. E.g. `dcos spark`. This change just makes it so that we don't need virtualenv for install the core DCOS CLI.